src/Fixture/Factory/PageFixtureFactory.php

```diff
@@ -85,6 +86,16 @@ public function create(array $options = []): PageInterface
         $page->setEnabled($options['enabled']);
         $page->setCode($options['code']);
 
+        $publishAt = $options['publish_at'] ?? null;
+        if ($publishAt) {
+            $page->setPublishAt(new DateTime($publishAt));
+        }
+
+        $unpublishAt = $options['unpublish_at'] ?? null;
+        if ($unpublishAt) {
+            $page->setUnpublishAt(new DateTime($unpublishAt));
+        }
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** if you add the normalizer, unnecessary conditions

```suggestion
        $page->setPublishAt($options['publish_at']);
        $page->setUnpublishAt($options['unpublish_at']);
```‏